.dccache

```diff
@@ -0,0 +1 @@
86bea98055f71b7e3a6cc63f5842391e577a8f25"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/pwd/pwd_linux_riscv64.go":[43737,1707755769748.014,"26c73f7f848b0775e26be6b5e894e6f2eae9382dcabf0ea715b11ca41aabbefd"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/pwd/pwd_linux_s390x.go":[35075,1707755769749.9436,"e6fddb968fa9a52ae96dbb426e3ae93a2e17dfa7ef9382bcbf471b3f2113118e"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/pwd/pwd_netbsd_amd64.go":[78546,1707755769751.9563,"b1f819ed2cd3b4ca3edb7413564b277fd93f92e31e240ae41b390839c415a63a"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/pwd/pwd_netbsd_arm.go":[77837,1707755769754.0815,"77cf78f058b857d9ce6144aee41850c7aaea3bc84634f31c2d7175585d4e6313"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/pwd/pwd_openbsd_386.go":[34613,1707755769755.9573,"1260579be6f1aaeafaeeecf602b8a14dbe5ac5382c29547ffd4761c26a057770"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/pwd/pwd_openbsd_amd64.go":[34848,1707755769758.0417,"f224d30b843210dadc3dd8f48c427b5c60aee00a8b0775646a0503094ce1a9c5"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/pwd/pwd_openbsd_arm64.go":[34794,1707755769760.0825,"ba023e8663d60b60c12500dd0bec571b3166b790d38995833b6a3676386188d5"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/signal/capi_darwin_amd64.go":[436,1707755769764.8901,"339ef9af60eb2e80dc31cdd80aef5d710769c92dd7962c630fdd19ef96c75178"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/signal/capi_darwin_arm64.go":[436,1707755769765.2551,"c71790b4f53ba791ab30f9b563c11424afdf992f95d1db49f19970a7cd53530d"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/signal/capi_freebsd_386.go":[356,1707755769765.6138,"aa4b5f7387fc538042396136ed06450a78423c6bc2bf9e846a7665de7d2af8af"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/signal/capi_freebsd_amd64.go":[358,1707755769767.877,"009d3f36f6cf5b3efd18de991c1bf888952ba4bb7c1380804ff52e9bcc2c2196"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/signal/capi_freebsd_arm.go":[356,1707755769768.3933,"abf627d561c99242989fd8eb215eb00695797810380749969da55fc2b11ab37e"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zendesk/vendor/modernc.org/libc/signal/capi_freebsd_arm64.go":[358,1707755769768.7996,"009d3f36f6cf5b3efd18de991c1bf888952ba4bb7c1380804ff52e9bcc2c2196"],"/Users/miguelchavez/Documents/go/src/github.com/mchavez/simera/baton-zende
```

Comment:
I think you don't want to commit this file.

Reply:
Fixed